pom.xml

```
@@ -236,7 +246,7 @@
     <dependency> <!-- see maskClasses config -->
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>20.0</version>
+      <version>30.0-jre</version>
```

```suggestion
      <version>30.1-jre</version>
```

? https://github.com/google/guava/releases/tag/v30.1